dotnet/src/SemanticKernel.IntegrationTests/CoreSkills/PlannerSkillTests.cs

```diff
@@ -167,7 +169,8 @@ public async Task CreatePlanShouldHaveIfElseConditionalStatementsAndBeAbleToExec
     }
 
     [Theory]
-    [InlineData("Start with a X number equals to the current minutes of the clock and remove 20 from this number until it becomes 0. After that tell me a math style joke where the input is X number + \"bananas\"",
+    [InlineData(
```

Comment:
I would add a test with the conditional set to false to see if we get a plan without <if> tags using the same goal.

Reply:
added.